main.go

```diff
@@ -143,6 +145,10 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&showVersion, "version", false, "Show current version and exit.")
 }
 
+// Add RBAC for the authorized diagnostics endpoint.
+// +kubebuilder:rbac:groups=authentication.k8s.io,resources=tokenreviews,verbs=create
+// +kubebuilder:rbac:groups=authorization.k8s.io,resources=subjectaccessreviews,verbs=create
```

Comment:
I don't see this new RBAC showing up in the generated config. Have I missed it?

Do you have a link to any documentation describing how this is used, btw?

Reply:
I stole it from https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4182/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R244-R245

And I think it comes from: https://github.com/kubernetes-sigs/cluster-api/pull/9264/files#diff-5f809e136425aaf67e69338ebe143aa06634a81c0632ddaed85894129029d85bR21-R32

Reply:
@mboersma maybe you have more context than I do to answer Matt's question. Thanks!

Reply:
It's probably not showing up in `rbac.yaml` because the `make generate-manifests` target isn't looking at `main.go`. 

https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/ca2516072b453fa4cefbb60452e3343e69317929/Makefile#L274-L281

Adding `paths=./ \` like [CAPI has](https://github.com/kubernetes-sigs/cluster-api/blob/main/Makefile#L281) should fix that. (I had made a similar change to CAPZ but must have lost it in refactoring; I'm glad you pointed this out.)

The docs on this feature in CAPI are here as part of PR 9264: https://github.com/kubernetes-sigs/cluster-api/pull/9264/files#diff-c60ebbe9702722c1666539c3ca1c30aaeafa37e5d420a62130752da313cf1156, not in the CAPI Book yet I assume because this isn't yet in a released version.

Reply:
Thanks a lot @mboersma ! :-)